moveit_ros/moveit_servo/test/test_servo_singularity.cpp

```diff
@@ -51,8 +51,8 @@ TEST_F(ServoFixture, ReachSingular)
   ASSERT_TRUE(start());
   EXPECT_EQ(latest_status_, moveit_servo::StatusCode::NO_WARNING);
 
-  // Look for DECELERATE_FOR_SINGULARITY status
-  watchForStatus(moveit_servo::StatusCode::DECELERATE_FOR_SINGULARITY);
```

Comment:
Would you mind adding a test that would check for the approach and hard stop conditions also?

Reply:
Lines 109 and lines 130